install/kubernetes/cilium/values.yaml

```diff
@@ -199,10 +199,20 @@ alibabacloud:
   # -- Enable AlibabaCloud ENI integration
   enabled: false
 
-# -- Optimize TCP and UDP workloads and enable rate-limiting traffic from
-# individual Pods with EDT (Earliest Departure Time)
-# through the "kubernetes.io/egress-bandwidth" Pod annotation.
-bandwidthManager: false
+# -- Deprecated in favor of bandwidthManager.enabled. To be removed in 1.13.
+# Optimize TCP and UDP workloads and enable rate-limiting traffic from
+# individual Pods with EDT (Earliest Departure Time) through the
+# "kubernetes.io/egress-bandwidth" Pod annotation.
+#bandwidthManager: false
+
+# -- Enable bandwidth manager to optimize TCP and UDP workloads and allow
+# for rate-limiting traffic from individual Pods with EDT (Earliest Departure
+# Time) through the "kubernetes.io/egress-bandwidth" Pod annotation.
+bandwidthManager:
+  # -- Enable bandwidth manager infrastructure (also prerequirement for BBR)
+  enabled: false
+  # -- Activate BBR TCP congestion control for Pods
+  bbr: false
```

Comment:
What if we were to add support for a third algorithm in the future? Wouldn't it make more sense to have a string instead of a bool, to set the algorithm directly? It would default to `cubic` but could be changed to `bbr`.

Reply:
I was thinking as well about this, but the user expectation might be misleading. This work and base setup (fq + bpf host routing to retain sk, etc) is specifically catered around BBR. In future once upstream, there could be BBRv2, but still tbd if it would make sense to have a probe where we either use v1 and v2 if kernel allows or force a specific version.